compiler/tools/fpp-format/src/main/scala/fpp-format.scala

```diff
@@ -28,17 +28,18 @@ object FPPFormat {
     result match {
       case Left(error) => {
         error.print
-        System.exit(1)
+        errorExit
```

Comment:
Why is the error handled here, inside the command, and not up a level like all the other tools?

Reply:
No particular reason, it's just the way it's written.